### PR TITLE
checkIfReceiverLocationIsValid function had been added

### PR DESCRIPTION
**Pull request by ralsalloum**
- checkIfReceiverLocationIsValid function had been added.
- subscriptionbyadmin API annotations had been added.
- calculateordersthatnotbelongtoanyfinancialdues replaced by command.

### CODE REVIEW

**Review by Yazan99sh — APPROVED** [review]

File: backend/src/Constant/CaptainFinancialSystem/CaptainFinancialSystem.php [hunk 1]
```diff
@@ -58,4 +58,6 @@ final class CaptainFinancialSystem
     const TARGET_SUCCESS_AND_INCREASE_INT = 2;
   
     const TARGET_FAILED_INT = 3;
+
+    const OK_RESULT_CONST = 'ok';
```
Comment (Kenan-Hussein) [inline backend/src/Constant/CaptainFinancialSystem/CaptainFinancialSystem.php:62]: No need to CONST in the end, this comment for the feature only

File: backend/src/Constant/Notification/NotificationFirebaseConstant.php [hunk 2]
```diff
@@ -123,4 +123,6 @@ final class NotificationFirebaseConstant
     const PLEASE_RECEIVE_DUES = 'لو سمحت بناء على طلبك تفضل باستلام مستحقاتك بتاريخ ';
     
     const CAPTAIN_ANSWER_DIFFERS_FROM_THAT_OF_STORE = 'يوجد اختلاف بتأكيد المتجر بالنسبة لتسديد مبلغ الطلب رقم';
+
+    const NOT_VALID_RECEIVER_LOCATION_OF_ORDER_CONST = 'لم يتم إدخال رابط موقع جغرافي صحيح من أجل الطلب ذو الرقم ';
```
Comment (Kenan-Hussein) [inline backend/src/Constant/Notification/NotificationFirebaseConstant.php:127]: this text must change to something better

File: backend/src/Service/Order/OrderService.php [hunk 3]
```diff
@@ -1478,4 +1481,20 @@ public function filterCashOrdersPaidOrNotByStore(CashOrdersPaidOrNotFilterByStor
 
         return $response;
     }
+
+    public function checkIfReceiverLocationIsValid(int $orderId, array $destination): void
+    {
+        if (count($destination) > 0) {
```
Comment (Kenan-Hussein) [inline backend/src/Service/Order/OrderService.php:1487]: please add those conditions too:
1- there is the distance that is calculated.
2- there is an order cost that is caculated.

**Review by Kenan-Hussein — COMMENTED** [review]
I left some comments

**Pull request merged**